### ISSUE TITLE
[PHP 7.3] compact() issues an E_NOTICE level error

### ISSUE THREAD

**Issue by drbyte** [ARCANEDEV/LogViewer#234]
- LogViewer Version: 4.6.0
- Laravel Version: 5.7.2
- PHP Version: 7.3.0-beta-2
- Hosted locally on Laravel Valet

### Description:
Trying to `compact` a non-defined variable.

https://github.com/ARCANEDEV/LogViewer/blob/a8cd768e8ab4f2ec80d2d3208f34af2799c847c2/src/Http/Controllers/LogViewerController.php#L82-L89

### Steps To Reproduce:
Followed @arcanedev-maroc instructions in https://github.com/ARCANEDEV/LogViewer/issues/195#issuecomment-367388830
```bash
composer create-project laravel/laravel fresh-laravel && cd fresh-laravel
composer require arcanedev/log-viewer
// set LOG_CHANNEL=daily
php artisan log-viewer:publish
```
Added logging to route:
```php
Route::get('/', function () {
    Log::info('This is info log');

    return view('welcome');
});
```
- Visit URL to create some logs
- Visit log-viewer URL: http://fresh-laravel.test/log-viewer
- Click on the "Logs" link.  Get error: `compact(): Undefined variable: footer`

<img width="1114" alt="screen shot 2018-09-10 at 9 11 42 pm" src="https://user-images.githubusercontent.com/404472/45332468-bee3cc80-b53e-11e8-9b7e-917498572b8f.png">

**Comment by drbyte** [comment]
Note: after swapping `footer` with `stats`, and `search` with `date`, the view still contains another `compact()` call to `query`, which doesn't exist either:

https://github.com/ARCANEDEV/LogViewer/blob/a8cd768e8ab4f2ec80d2d3208f34af2799c847c2/resources/views/bootstrap-4/show.blade.php#L163

**Comment by arcanedev-maroc** [comment]
This is why:
>Note:
>Before PHP 7.3, any strings that are not set will silently be skipped.

### Changelog 

Version | Description
-- | --
7.3.0 | compact() now issues an E_NOTICE level error if a given string refers to an unset variable. Formerly, such strings have been silently skipped.

Source: https://secure.php.net/manual/en/function.compact.php

**PHP 7.3 is not supported for the time being but it's good to know** :+1:

**Comment by HDVinnie** [comment]
Just replace usages of `compact()` with `with()`
```
return $this->view('logs', compact('headers', 'rows', 'footer'));
```
TO
```
return $this->view('logs', ['headers' => $headers, 'rows' => $rows, 'footer' => $footer]);
```
OR
```
return $this->view('logs')->with('headers' => $headers, 'rows' => $rows, 'footer' => $footer);
```

**Comment by drbyte** [comment]
@arcanedev-maroc no, this isn't a PHP 7.3 issue.

It's bugs that were created in #153 when refactoring was done which relocated some variables but didn't update the variables passed to `compact`, and changes were made to views.

See PR #242

**Comment by arcanedev-maroc** [comment]
Patch released: `v4.6.1`
